v3iofs/fs.py

```diff
@@ -176,14 +187,15 @@ def _open(
         mode='rb',
         block_size=None,
         autocommit=True,
-        cache_options=None,
+        cache_options="readahead",
```

Comment:
What does this do?

Reply:
fsspec provides several caching methods.  The default is readahead.  See:  https://github.com/intake/filesystem_spec/blob/master/fsspec/caching.py